packages/react-data-grid/src/masks/InteractionMasks.js

```diff
@@ -404,20 +416,23 @@ class InteractionMasks extends React.Component {
   };
 
   selectCell = (cell, openEditor) => {
-    const callback = openEditor ? this.openEditor : undefined;
-    const next = { ...this.state.selectedPosition, ...cell };
-    if (this.isCellWithinBounds(next)) {
-      this.setState({
-        selectedPosition: next,
-        selectedRange: {
-          topLeft: next,
-          bottomRight: next,
-          startCell: next,
-          cursorCell: next,
-          isDragging: false
-        }
-      }, callback);
-    }
+    const callback = openEditor ? this.openEditor : () => null;
```

Comment:
Just so I understand, what is the reason for callback change from `undefined` to `() => null`

Reply:
selectCell was being called from other places without passing in a second parameter. This was generating a script error so I added a default callback which does nothing